src/wpt/harness.ts

```diff
@@ -140,7 +190,7 @@ globalThis.assert_implements = (condition, description) => {
  */
 globalThis.assert_implements_optional = (condition, description) => {
   if (!condition) {
-    throw new OptionalFeatureUnsupportedError(description);
+    throw new OptionalFeatureUnsupportedError({ message: description });
```

Comment:
Since it extends AssertionError, the first parameter needs to be a string. I don't follow why this change was necessary.

Reply:
I am a bit puzzled too, but TS is telling me: `error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type '{ message?: string | undefined; actual?: unknown; expected?: unknown; operator?: string | undefined; stackStartFn?: Function | undefined; } | undefined`

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5250f7a4bd6d7ff68f0ebb93da35f3b75aa22bdc/types/node/assert.d.ts#L38

Reply:
It seems web-platform-tests implementation of AssertionError is different from Node.js - https://github.com/web-platform-tests/wpt/blob/038c19b8529b374bb40b6b01cf062577f0dc39dc/resources/testharness.js#L4554 - Can you add a comment that this is indeed correct, and it is different from WPT AssertionError.

Reply:
Actually this means we shouldn't use AssertionError from Node.js and have a similar API just like WPT. Because some tests might call this error with a string as a parameter.